packages/office-ui-fabric-react/src/components/DetailsList/examples/DetailsList.Basic.Example.tsx

```diff
@@ -68,6 +68,7 @@ export class DetailsListBasicExample extends React.Component<{}, IDetailsListBas
         />
         <MarqueeSelection selection={this._selection}>
           <DetailsList
+            checkButtonAriaLabel="Row checkbox"
```

Comment:
Is there a reason you only added a `checkButtonAriaLabel` value to the basic example? :)

Reply:
Added on every example.